packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3681,7 +3681,7 @@ class Flex extends MultiChildRenderObjectWidget {
        assert(mainAxisSize != null),
        assert(crossAxisAlignment != null),
        assert(verticalDirection != null),
-       assert(crossAxisAlignment != CrossAxisAlignment.baseline || textBaseline != null),
+       assert(!identical(crossAxisAlignment, CrossAxisAlignment.baseline) || textBaseline != null),
```

Comment:
Maybe just with a TODO linking to https://github.com/dart-lang/sdk/issues/29278 ?  That's marked as something that will be fixed someday.

Reply:
It's kinda moot since I can't merge this PR with the commented-out assert in `MultiChildRenderObjectWidget`.

https://github.com/dart-lang/sdk/issues/29276

Reply:
Ahh ok